src/app/core/services/qbo/qbo-configuration/qbo-connector.service.ts

```
@@ -38,6 +38,11 @@ export class QboConnectorService {
     return this.apiService.get(`/workspaces/${this.workspaceService.getWorkspaceId()}/credentials/qbo/`, {});
   }
 
+  @Cacheable()
+  getQboTokenHealth(): Observable<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)